docs/reference/sql/data-types.md

```diff
@@ -303,24 +309,38 @@ INSERT INTO bools(b) VALUES (TRUE), (FALSE);
-| `UInt16`               | `UnsignedSmallInt`                                              |
-| `UInt32`               | `UnsignedInt`                                                   |
-| `UInt64`               | `UnsignedBigInt`                                                |
-| `Float32`              | `Float`                                                         |
-| `Float64`              | `Double`                                                        |
-| `TimestampSecond`      | `Timestamp_s`, `Timestamp_sec`, `Timestamp(0)`                  |
-| `TimestampMillisecond` | `Timestamp`, `Timestamp_ms`, `Timestamp(3)`                     |
-| `TimestampMicroSecond` | `Timestamp_us`, `Timestamp(6)`                                  |
-| `TimestampNanosecond`  | `Timestamp_ns`, `Timestamp(9)`                                  |
+
+| SQL Datatype Alias                                                            | Native Datatype          |
+| --------------------------------------------------------------- | ---------------------- |
+| `Text`, `TinyText`, `MediumText`, `LongText`, `Varchar`, `Char` | `String`               |
+| `Varbinary`                                                     | `Binary`               |
+| `TinyInt`                                                       | `Int8`                 |
+| `SmallInt`, `Int2`                                              | `Int16`                |
+| `Int`, `Int4`                                                   | `Int32`                |
+| `BigInt`, `Int8`                                                | `Int64`                |
+| `UnsignedTinyInt`                                               | `UInt8`                |
+| `UnsignedSmallInt`                                              | `UInt16`               |
+| `UnsignedInt`                                                   | `UInt32`               |
+| `UnsignedBigInt`                                                | `UInt64`               |
+| `Float`, `Float4`                                               | `Float32`              |
+| `Double`, `Float8`                                              | `Float64`              |
+| `Timestamp_s`, `Timestamp_sec`, `Timestamp(0)`                  | `TimestampSecond`      |
+| `Timestamp`, `Timestamp_ms`, `Timestamp(3)`                     | `TimestampMillisecond` |
+| `Timestamp_us`, `Timestamp(6)`                                  | `TimestampMicroSecond` |
+| `Timestamp_ns`, `Timestamp(9)`                                  | `TimestampNanosecond`  |
+
+:::warning Breaking Change
+The type aliases `Int2`, `Int4`, `Int8`, `Float4`, and `Float8` follow the PostgreSQL and MySQL convention where these identifiers refer to the number of **bytes** (not bits) in the type.
+
+Specifically:
+- `Int2` = 2 bytes = `SmallInt` (16-bit)
+- `Int4` = 4 bytes = `Int` (32-bit)
+- `Int8` = 8 bytes = `BigInt` (64-bit) - **Breaking**: Previously mapped to `TinyInt` (8-bit)
```

Comment:
No need to mention breaking change either. Otherwise we should specify break from which version.